src/__tests__/remove-non-beta-product-pages.test.ts

```diff
@@ -0,0 +1,35 @@
+import { execFileSync } from 'child_process'
+
+describe('remove-non-beta-product-pages', () => {
+	test('only removes expected page directories', () => {
+		let stdout = String(
+			execFileSync(
+				'npx',
+				['hc-tools', './scripts/remove-non-beta-product-pages.ts'],
+				{
+					env: {
+						...process.env,
+						HASHI_ENV: 'production',
+						CI: '1',
+						DRY_RUN: '1',
+					},
+				}
+			)
+		)
+
+		stdout = stdout.split('\n').slice(1).join('\n')
+
+		expect(stdout).toMatchInlineSnapshot(`
+		"🧹 removing pages at /boundary
+		🧹 removing pages at /consul
+		🧹 removing pages at /docs
+		🧹 removing pages at /hcp
+		🧹 removing pages at /nomad
+		🧹 removing pages at /packer
+		🧹 removing pages at /sentinel
+		🧹 removing pages at /terraform
+		🧹 removing pages at /vagrant
+		"
+	`)
```

Comment:
This snapshot test will fail anytime the output changes, which should be a good signal for us to manually review (we will have to anyways in order to get tests to pass).